website/source/docs/enterprise/replication/index.html.md

```diff
@@ -67,10 +67,10 @@ They do not forward service read or write requests until they are elected and be
 
 | Capability                                                                                                               	| Disaster Recovery 	| Performance                                                              	|
 |--------------------------------------------------------------------------------------------------------------------------	|-------------------	|--------------------------------------------------------------------------	|
-| Mirrors the secrets infrastructure of a primary cluster                                                                  	| Yes               	| Yes                                                                      	|
-| Mirrors the configuration of a primary cluster’s backends (i.e.: auth methods, storage backends, secrets engines, etc.) 	| Yes               	| Yes                                                                      	|
-| Contains a local replica of secrets on the secondary and allows the secondary to forward writes                          	| No                	| Yes                                                                      	|
-| Mirrors the token auth infrastructure for applications or users interacting with the primary cluster                     	| Yes               	| No. Upon promotion, applications must re-auth tokens with a new primary. 	|
+| Mirrors the configuration of a primary cluster                                                                  	| Yes               	| Yes                                                                      	|
+| Mirrors the configuration of a primary cluster’s backends (i.e.: auth methods, secrets engines, audit devices, etc.) 	| Yes               	| Yes                                                                      	|
+| Allows the secondary cluster to forward writes                          	| No                	| Yes                                                                      	|
```

Comment:
Since it's transparent to the user and generally not necessary to understand about the whole write-forwarding thing, I actually think that we should change this to say "Allows the secondary cluster to handle client requests" and would put this below the next line. That way the logical flow is "hm, performance doesn't mirror the tokens/leases, why would I want that...OH I see".